woocommerce/woocommerce-ios
# SPE: Hide the reviews option from product details, when there are none

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8841

## Description

This PR hides "Reveiws" row on product details form even when there are no reviews on the product.

## Testing

1. Build and run the app in debug/alpha mode.
2. Open product details for a product without any reviews (or start product creation flow).
3. Confirm "Reviews" cell on the product form is hidden.
4. Tap "Add more details" on the bottom of the screen.
5. Confirm "Reviews" cell is displayed in the action sheet.
6. Tap it and confirm navigation works.
7. Open different product with existing reviews.
8. Confirm "Reviews" cell is displayed on the product form.

## Screenshots

before|after
--|--
![Simulator Screen Shot - 1](https://user-images.githubusercontent.com/3132438/220065012-db7fff3b-9b26-4bfe-84de-90f0c58744c8.png)|![Simulator Screen Shot - 3](https://user-images.githubusercontent.com/3132438/220065556-8d57b1e1-c5f8-4eab-b58d-17a239f62def.png)
![Simulator Screen Shot - 2](https://user-images.githubusercontent.com/3132438/220065018-b7be9198-b95b-48bf-8d1f-4d3dc6fdba34.png)|![Simulator Screen Shot - 4](https://user-images.githubusercontent.com/3132438/220065584-7dcae0d6-6528-402d-9f82-930db115313f.png)

---
